apps/test/unit/code-studio/pd/application_dashboard/quick_viewTest.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import QuickView from '@cdo/apps/code-studio/pd/application_dashboard/quick_view';
+import {expect} from 'chai';
+import sinon from 'sinon';
+
+describe("Quick View", () => {
+  const fakeRouter = {
+    createHref() {}
+  };
+
+  const context = {
+    router: fakeRouter
+  };
+
+  it("Initially renders a spinner", () => {
+    let quickView = shallow(
+      <QuickView route={{'path':'csf_facilitators', 'title':'CSF Facilitators'}}/>, {context}
+    );
+
+    expect(quickView.find('Spinner')).to.have.length(1);
+  });
+
+  it("Generates 1 table after hearing from server", () => {
+    let server = sinon.fakeServer.create();
+
+    server.respondWith("GET", '/api/v1/pd/applications/quick_view?role=csf_facilitators',
+      [
+        200,
+        {"Content-Type": "application/json"},
+        JSON.stringify([{"id":8,"created_at":"2017-10-25T21:26:06.000Z","applicant_name":"Clare Constantine","district_name":null,"school_name":null,"status":"unreviewed"}])
```

Comment:
style nit: blow up the object, one key per line for readability. It'll probably be more readable if created above as a const instead of inline this deep